src/matchers/toBeNaN/predicate.js

```diff
@@ -0,0 +1 @@
+export default expected => isNaN(expected);
```

Comment:
I think this should just check that the value is strictly `NaN` (===).
Something like:

```js
export default expected => expected === NaN;
```